README.md

```diff
@@ -10,10 +10,10 @@ There are currently some workarounds needed to get the API Mapper up and running
 
 1. Make the newly created service public
 ![Public Service](/public/images/publicservice.jpg)
-2. Visit the data browser, and upgrade the database so this application has it's own database. Note the application needs to be first finished its initial deploy and be running to perform this task. 
+2. Visit the data browser, depending on your app configuration a "Upgrade Database" action will be available, this means the application has its own database and it is a **mandatory step** to upgrade it. Note the application needs to be first finished its initial deploy and be running to perform this task. 
```

Comment:
I wonder if the wording is accurate here?

- if the 'upgrade database' option is shown, it means the app is using an old style database (behind fh-ditch), and **should** be upgraded (so the right env vars are set for the app)
- if the 'upgrade database' option is **not** shown, it means the app has its own dedicated database already (and the right env vars are already set for the app), so no action is required by the user